Example/Tests/Tests-Info.plist

```diff
@@ -7,13 +7,11 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleExecutable</key>
-	<string>$(EXECUTABLE_NAME)</string>
+	<string>${EXECUTABLE_NAME}</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.tumblr.$(PRODUCT_NAME:rfc1034identifier)</string>
+	<string>org.cocoapods.demo.${PRODUCT_NAME:rfc1034identifier}</string>
```

Comment:
Hm. This is what `pod lib create` gives out of the box. Not sure if it matters but guessing people don't generally change it?